### PR TITLE
[Event] Created hyperlinks only for the speaker with profiles.

### LINKED ISSUE #1762: [Event] KubeCon EU: Multiple speakers hyperlinked to same member profile
**Description**
The hyperlink to speakers on [this event](https://layer5.io/community/events/cncf-sig-network-intro-and-deep-dive) is broken in that it points to two people on the same member profile. Member profiles are individually hosted.

Ken does not have a profile.
Lee has a profile.

**Expected Behavior**
Either no hyperlink at all or only Lee's name hyperlinked.

**Screenshots**
Screen Shot 2021-04-05 at 2 48 25 PM

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by SachiniAbeygunawardhana**
**Description**
[Event] Created hyperlinks only for the speaker with profiles. 

This PR fixes #1762

**Notes for Reviewers**

1. The issue with the hyperlinks is fixed.
2. The event description is added

Please let me know if any further improvement is needed.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [yes] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit a1749673d6b784acfacff9ab75467750601f402a at: https://607138209bf93387876e3cb3--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 59dce8b916090da543c22c7059f2b8685fd124ec at: https://60713a4d15b6ef98d857598a--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 59dce8b916090da543c22c7059f2b8685fd124ec at: https://60713d56041835a8caf170bf--layer5ng.netlify.app

File: src/collections/events/kubecon-eu-2021/index.mdx [hunk 1]
```diff
@@ -5,6 +5,8 @@ thumbnail: ./kubecon-europe-2021.jpg
 eurl: https://kccnceu2021.sched.com/event/ijVr/cncf-sig-network-intro-and-deep-dive-lee-calcote-layer5-ken-owens-fiserv
 topic: event
```
Comment (Tanuj22) [inline src/collections/events/kubecon-eu-2021/index.mdx:6]: Can you also add `upcoming: true`  and `feature: true` here. And remove it from [here](https://github.com/layer5io/layer5/blob/master/src/collections/events/advanced-istio/index.mdx)

**Review by Tanuj22 — COMMENTED** [review]
@SachiniAbeygunawardhana everthing looks good. Just one addition and we are good to go

File: src/collections/events/kubecon-eu-2021/index.mdx (hunk 1 above)
Comment (SachiniAbeygunawardhana) [reply src/collections/events/kubecon-eu-2021/index.mdx:6]: sure, will do

**Comment by l5io** [comment]
🚀 Preview for commit 3aa961ba43d5ad0d68524145eb2995c20dd0f7e4 at: https://60733bc7121793f11b58b28d--layer5ng.netlify.app

**Review by Tanuj22 — APPROVED** [review]
Thank you @SachiniAbeygunawardhana

**Pull request merged**